config/pipelines/high_throughput_scrna_core_cdna_prep.wip.yml

```diff
@@ -26,3 +26,24 @@ scRNA Core cDNA Prep Input:
     LRC HT 5p Chip: LRC HT 5p GEMs
     LRC HT 5p GEMs: LRC HT 5p cDNA PCR
     LRC HT 5p cDNA PCR: LRC HT 5p cDNA PCR XP
+scRNA Core Chromium HT Aggregation:
+  pipeline_group: scRNA Core cDNA Prep
```

Comment:
Can we have a new 'pipeline group' called 'scRNA Core Library Prep'? We're out of the 'cDNA Prep' phase here. (Or, two new ones - one for aggregation and one for library prep)

They're just string values so just need to change it here, and in the below pipeline.

Reply:
yes will change.
also wondered if they should be in separate config files, but will leave together for now, we can discuss.